static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js

```diff
@@ -71,7 +73,33 @@ define([
         }
     }
 
+
+
+    function shouldAutoplay(){
+
+        function isMobile() {
+            const isIOS = () => /(iPad|iPhone|iPod touch)/i.test(navigator.userAgent);
+            const isAndroid = () => /Android/i.test(navigator.userAgent);
+
+            return isIOS() || isAndroid();
```

Comment:
is this really the check you want? isMobile doesn't really seem to equal being iOS or android does it? do you want a breakpoint maybe?